### PR TITLE
Sepseleva / Add Task7

### PR DESCRIPTION
**Pull request by MariShi91**

### CODE REVIEW

File: Tasks/Sepseleva/Task7/data.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import copy
+from decimal import Decimal
+repo = {
+    12341:{"name":"Harry Potter","author":"J.K. Rowling", "price": '36.99', "category":"Fantasy", "number":5},
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Sepseleva/Task7/data.py:4]: Harry Potter and what exactly?))

File: Tasks/Sepseleva/Task7/data.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                value = int(v['number']) - int(number)
+                if value >= 0:
+                    v["number"] = value
+                    create_new_item(k, number)
+                    return True
+                else:
+                    number = v['number']
+                    v["number"] = 0
+                    create_new_item(k, number)
+                    raise RuntimeError(f"Only {number} items left in stock. They've been added to the basket")
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Sepseleva/Task7/data.py:42]: this is good choice

File: Tasks/Sepseleva/Task7/bl.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    return "There's no such a book in our store"
+    
+def show_basket():
+    res = data.show_basket()
+    t = show_contents(res)
+    if t:
+        return f"""
+        ***Items in your basket***
+{t}"""
+    return "Your basket is empty now. To add items input '2'"
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Sepseleva/Task7/bl.py:36]: do not provide a user with UI suggestions from bl, these two layers should know nothing about each other

**Review by MikitaTsiarentsyeu — APPROVED** [review]
this is a decent job, I grant you 9 points

File: Tasks/Sepseleva/Task7/data.py (hunk 1 above)
Comment (MariShi91) [reply Tasks/Sepseleva/Task7/data.py:4]: Haha) Shame on me :) , Actually, I couldn't choose one and didn't want to add all of them here) Let it be "Harry Potter and the Deathly Hallows: Part 1")

File: Tasks/Sepseleva/Task7/bl.py (hunk 3 above)
Comment (MariShi91) [reply Tasks/Sepseleva/Task7/bl.py:36]: ThankQ, I got it)

Comment (MariShi91) [reply Tasks/Sepseleva/Task7/bl.py:36]: ThanQ, I got it)

**Pull request merged**